src/v0/destinations/redis/transform.js

```diff
@@ -46,6 +46,19 @@ const transformSubEventTypeProfiles = (message, workspaceId, destinationId) => {
   };
 };
 
+const getJSONValue = (message) => {
+  const eventType = message.type.toLowerCase();
+  if (eventType === EventType.IDENTIFY) {
+    return getFieldValueFromMessage(message, 'traits');
+  }
+  return {};
```

Comment:
So for non-identify events, JSONValue is returned as {} and gets stored? or is the empty object eventually discarded at redis level?

Reply:
Non-identify events would not actually hit this method as we have a [validation](https://github.com/rudderlabs/rudder-transformer/pull/3336/files#diff-9b24d1e8d6b9bdeaf54ae33caa14dd259141f74bdcd224ef555f42e50de5bdb9L53-L55) in place.